content/docs/command-reference/import.md

```diff
@@ -104,9 +105,10 @@ repo at `url`) are not supported.
   > example below).
 
 - `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
```

Comment:
I think `url` is not the best term here (mind the difference in the command names - import vs import-_url_)

Reply:
Right, that makes sense since `dvc import` handles repo locations. Would `repo url` fit much better here?

Moreover, if we do replace the `url` with something else, we'll also need to update the CLI parser (_like the screenshot below_) as well as the rest of the doc for consistency.

![image](https://user-images.githubusercontent.com/3449761/97841416-bf441080-1d20-11eb-80d3-56b7377de682.png)